madmom/utils/__init__.py

```diff
@@ -419,6 +426,97 @@ def quantize_events(events, fps, length=None, shift=None):
     return quantized
 
 
+def quantize_notes(notes, fps, length=None, num_notes=None, shift=None,
+                   offset=None, velocity=None):
+    """
+    Quantize the notes with the given resolution.
+
+    Parameters
+    ----------
+    notes : 2D numpy array
+        Notes to be quantized.
+    fps : float
+        Quantize with `fps` frames per second.
+    length : int, optional
+        Length of the returned array. If 'None', the length will be set
+        according to the latest sounding note.
+    num_notes : int, optional
```

Comment:
The name of this parameter is easily confused with the "number of notes in the notes array". Maybe changing the description to something like "Number of possible pitches"? Not sure...

Reply:
Renamed to `num_pitches`.